package.json

```diff
@@ -46,13 +46,15 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.29.4",
     "eslint-plugin-react-hooks": "^4.5.0",
-    "eslint-plugin-testing-library": "^5.3.1",
-    "prettier-eslint": "^13.0.0",
```

Comment:
Prettier ESLint requires a major upgrade of `eslint`. I'm not sure every repository is ready for it yet. I guess we'll find out :man_shrugging: